test/lib/benchmark.js

```diff
@@ -6,9 +6,8 @@
 'use strict'
 
 const helper = require('./agent_helper')
-const async = require('async')
 
-exports.createBenchmark = function createBenchmark(opts) {
+exports.createBenchmark = (opts) => {
```

Comment:
was this change actually needed?

Reply:
This one...not in terms of functionality. I've been converting function definitions to arrow functions where I can, so this was stylistically in keeping with that.